packages/manager/apps/web/client/app/domain/options/delete/domain-options-delete.controller.js

```diff
@@ -0,0 +1,47 @@
+angular.module('controllers').controller(
+  'controllers.Domain.Options.Delete',
+  class DomainDnsLockCtrl {
+    constructor($rootScope, $scope, $translate, Alerter, Domain) {
```

Comment:
```suggestion
    /* @ngInject */
    constructor($rootScope, $scope, $translate, Alerter, Domain) {
```

Reply:
Not part of this PR. It's just a `git subtree pull` from the original `develop` branch (https://github.com/ovh-ux/ovh-manager-web).